composer/modules/web/scss/modules/ballerina-editor.scss

```diff
@@ -1,296 +1,444 @@
-                    fill: none;
-                    stroke: #777;
-                    stroke-dasharray: 3,3;
-                }
-                .flowchart-separator-line {
-                    stroke: #f9f9f9;
-                    stroke-width: 2;
-                }
-                .flowchart-true-text {
-                    text-anchor: end;
-                    font-style: italic;
-                    font-weight: bolder;
-                }
-                .flowchart-false-text {
-                    text-anchor: start;
-                    font-style: italic;
-                    font-weight: bolder;
-                }
-                .property-pane-action-button-wrapper{
-                    fill: #e2e2e2;
-                    padding: 10px;
-                }
-                .property-pane-action-button-delete, .property-pane-action-button-jump{
-                    opacity: 0.6;
-                    &:hover{
-                        opacity: 1;
-                    }
-                }
-            }
+          }
+        }
+
+        .parameter-text, .parameter-bracket-text, .return-types-opening-brack-text, .parameter-bracket-text {
+          fill: #333;
+          dominant-baseline: text-before-edge;
+          cursor: default;
         }
+      }
     }
+  }
+  .bottom-right-controls-container {
+    position: absolute;
+    bottom: 20px;
+    right: 50px;
+    z-index: 30;
+    color: #f9f9f9;
+    padding: 0;
+    line-height: 30px;
+    width: 120px;
 
-    .panel-body{
-        .panel {
-            .panel-header, .svg-container {
-                fill: #f9f9f9;
-                .headingRect {
-                    stroke-width: 1px;
-                    stroke: #e4e4e4;
-                    .panel-header-controls {
-                        .panel-header-controls-wrapper {
-                            fill: rgba(255, 255, 255, 0.1);
-                        }
-                    }
+    .bottom-label-icon-wrapper {
+      height: 30px;
+      width: 30px;
+      background-color: #e8e8e8;
+      margin: 0 5px;
+      float: left;
+      padding: 2px 8px;
+      border: 1px solid #c1c1c1;
+      color: #333;
+    }
 
+    .bottom-label-icon-wrapper:hover {
+      position: relative;
+      cursor: pointer;
+    }
+
+    .bottom-label-icon-wrapper:before {
+      content: attr(title);
+      position: absolute;
+      top: -3em;
+      left: -1em;
+      visibility: hidden;
+      color: #fff;
+      text-align: center;
+      background-color: #303030;
+      width: 100px;
+    }
+
+    .bottom-label-icon-wrapper:hover:after {
+      border: 11px solid transparent;
+      border-top-color: #303030;
+      content: '';
+      display: block;
+      height: 0;
+      width: 0;
+      position: absolute;
+      top: -14px;
+      font-size: 12px;
+    }
+
+    .bottom-label-icon-wrapper:hover:before {
+      visibility: visible;
+    }
+
+    .bottom-view-label {
+      display: none;
+    }
+  }
+}
+
+.welcome-page {
```

Comment:
Shall we move the welcome styles to a different sheet ?

Reply:
Sure will do